membrane_rtc_engine_hls/test/support/file_source_endpoint.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Membrane.RTC.Engine.Support.FileEndpoint do
+defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
```

Comment:
Why moved? I thought we agreed not to move it and instead create fake source just for engine :thinking: Also, formatting shouldn't change

Reply:
But it is as it was in HLS I only rename it.